src/database/models.py

```diff
@@ -4022,6 +4175,180 @@ def full_json(self):
         data["community"] = self.community.info()
         data["updated_by"] = self.updated_by.info() if self.updated_by else None
         return data
-    
+
     class Meta:
         indexes = [ models.Index(fields=["community", "notification_type"]),]
+
+    class TranslationMeta:
+        fields_to_translate = ["notification_type"]
+
+
+# localisation
+class SupportedLanguage(BaseModel):
+    """
+    A class used to represent the languages supported by the platform
+
+    Attributes
+    ----------
+    name : str
+      name of the language.
+    code : str
+
+    """
+
+    code = models.CharField(max_length=LANG_CODE_STR_LEN, unique=True)
+    name = models.CharField(max_length=SHORT_STR_LEN, unique=True)
+    is_rtl = models.BooleanField(default=False, blank=True) # not used now but maybe used in the future
```

Comment:
what is rtl?  right to left language? or left to right?

Reply:
Right to left language..
I've changed it to the full meaning to remove any confusion.